src/home/current-positions/components/position-controls/index.tsx

```diff
@@ -0,0 +1,303 @@
+import * as React from 'react';
+import find from 'lodash/find';
+import Button from 'common/button';
+import Typography from '@mui/material/Typography';
+import styled from 'styled-components';
+import { FormattedMessage } from 'react-intl';
+import { NetworkStruct, Position, Token, YieldOptions } from 'types';
+import { useHistory } from 'react-router-dom';
+import { NETWORKS, OLD_VERSIONS } from 'config/constants';
+import { BigNumber } from 'ethers';
+import { buildEtherscanTransaction } from 'utils/etherscan';
+import OpenInNewIcon from '@mui/icons-material/OpenInNew';
+import Link from '@mui/material/Link';
+import { getWrappedProtocolToken, PROTOCOL_TOKEN_ADDRESS } from 'mocks/tokens';
+import IconButton from '@mui/material/IconButton';
+import MoreVertIcon from '@mui/icons-material/MoreVert';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import { withStyles } from '@mui/styles';
+import { createStyles } from '@mui/material/styles';
+import useWalletService from 'hooks/useWalletService';
+import { useAppDispatch } from 'state/hooks';
+import { setPosition } from 'state/position-details/actions';
+
+const StyledCardFooterButton = styled(Button)``;
```

Comment:
There is none, I think this was mostly a routine that I had, so I have more context on what that button is instead of just rendering a "Button" I can quickly know that its the "Card Footer Button"